playbook/app/pb_kits/playbook/utilities/globalProps.ts

```diff
@@ -180,9 +181,21 @@ const PROP_CATEGORIES: {[key:string]: (props: {[key: string]: any}) => string} =
     css += lineHeight ? `line_height_${lineHeight} ` : ''
     return css
   },
-  displayProps: ({ display }: Display) => {
+  displayProps: ( display: Display ) => {
     let css = ''
-    css += display ? `display_${display} ` : ''
+    Object.entries(display).forEach((displayEntry) => {
+      if (displayEntry[0] == "display") {
+        if (typeof displayEntry[1] == "string") {
```

Comment:
In reference to the comment above ^
```suggestion
    Object.entries(display).forEach((displayEntry) => {
        if (typeof displayEntry[1] == "string") {
```